profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestAccountTimeline.java

```diff
@@ -275,10 +275,8 @@ private void verifyBundles(final UUID accountId, final DateTime startTime, final
                 verifyAuditLog(subscriptionEvent2AuditLogs.get(0), ChangeType.INSERT, reason, comment, createdBy, startTime, endTime);
             } else {
                 // Audits for bundles
-                Assert.assertEquals(bundleAuditLogs.size(), 3);
+                Assert.assertEquals(bundleAuditLogs.size(), 1);
                 verifyAuditLog(bundleAuditLogs.get(0), ChangeType.INSERT, reason, comment, createdBy, startTime, endTime);
-                verifyAuditLog(bundleAuditLogs.get(1), ChangeType.UPDATE, null, null, TRANSITION, startTime, endTime);
```

Comment:
Do you understand why *all* updates have disappeared? I would have expected the code change to merge some, but not to make all of them disappear (1 update instead of multiple ones).

Reply:
Yep, I understand, this is because we don't update the [bundle#lastSysUpdateDate](https://github.com/killbill/killbill/pull/1583/files#diff-fa76b2b89459501b1c3442c56007ecc4a5c6f0aee2085469347119267b73e6e4L523)

Reply:
As mentioned in my [comment](https://github.com/killbill/killbill/pull/1583/commits/d4d12979a1e204aa1a73872e905d7efb213debcb) we should even remove the column in 0.23.x